collectors/bzimport/tests/test_convertors.py

```diff
@@ -89,6 +89,8 @@ def get_history(self):
                 impact=FlawImpact.IMPORTANT,
                 title="historical title",
                 description="historical description",
+                pgh_created_at=timezone.now(),
+                pgh_label="user",
```

Comment:
?

Reply:
The test does not pass without it now but I have not dig into it very much as the history functionality is not used. The difference is that there used to be just `clean` while now it is `full_clean` so some of the additional steps requires the values to be set - not sure why as it is `null=True` but not worth to investigate now as the history might be eventually completely different.

Reply:
ok, let's make a mental note of it and check again when we get rid of pg_history